sigs/cncf-sigs.md

```diff
@@ -0,0 +1,250 @@
+
+### Elections
+
+* The TOC nominates Chairs
+
+* Chairs are assigned following a 2/3 majority vote of the TOC
+
+* Terms last for 2 years but staggered such that at least 1 of the chairs is able to maintain continuity
+
+* The TOC and Chairs nominate Tech leads
+
+* Tech leads are assigned following a 2/3 majority vote of the TOC and a 2/3 majority vote of SIG Chairs
+
+* SIG Chairs and Tech Leads may be unassigned from the SIG at any time following a 2/3 majority vote of the TOC
+
+### Governance
+
+* All SIGs inherit and follow the CNCF TOC Operating Principles.
+
+* SIGs must have a documented governance process that encourages community participation and clear guidelines to avoid biased decision-making.  
+
+    * NOTE: aim here is to align with "minimal viable" model of the CNCF projects, and only have such governance as is needed, not anything too burdensome
+
+* They may grow a set of practices over time in the same way as an OSS Project, provided this is consistent with CNCF Operating Principles. 
+
+* As with CNCF Projects all exceptions and disputes are handled by TOC with CNCF Staff help
+
+### Budget & Resource
+
+* No formal systematic budget at this time, other than commitment of CNCF executive staff to provide named person as liaison point.
+
+* Just as CNCF Projects may have "help" offered by CNCF and may ask for things via the [ServiceDesk](https://github.com/cncf/servicedesk), the SIGs may do this.
+
+## Retirement
+
+* In the event that a SIG is unable to regularly establish quorum, or fulfill the responsibilities and/or regularly report to the TOC, the TOC will:
+
+    * Consider retiring the SIG after 3 months
+
+    * Must retire the SIG after 6 months
+
+* The TOC may, by means of a 2/3 majority vote, declare "no confidence" in the SIG.  In this event, the TOC may then vote to retire or reconstitute the SIG.
+
+## Initial SIGS
+
+To bootstrap the process, the TOC proposes the following SIGs, and projects assigned to each SIG. Clearly all of these SIG’s will not be fully-formed overnight or begin operating immediately, so the TOC itself will fulfill the duties of not-yet-formed SIG’s until they are.  We can however, fairly immediately, assign one voting member of the TOC as liason for each SIG, and prioritize the order of formation of the SIGs, starting immediately with the most pressing ones. 
+
+<table>
+  <tr>
+    <td>Name (to be finalised)</td>
+    <td>Area</td>
+    <td>Current CNCF Projects</td>
+  </tr>
+  <tr>
+    <td>Traffic</td>
+    <td>networking, service discovery, load balancing, service mesh, RPC, pubsub, etc.</td>
+    <td>Envoy, Linkerd, NATS, gRPC, CoreDNS, CNI</td>
+  </tr>
+  <tr>
+    <td>Observability</td>
+    <td>monitoring, logging, tracing, profiling, etc.
+</td>
+    <td>Prometheus, OpenTracing, Fluentd, Jaeger, Cortex, OpenMetrics, </td>
+  </tr>
+  <tr>
+    <td>Governance</td>
```

Comment:
```suggestion
    <td>Security & Compliance</td>
```

Reply:
@izgeri As mentioned in the intro, the names are placeholders.  I would prefer the renaming exercise to be undertaken holistically by the chosen SIG Chairs and TOC Liaison, once they have nailed down the scope more precisely through community consultation. One objection I can imagine to the alternative name you've proposed is that the two items you include in it do not adequately cover the intended scope of the SIG (e.g. policy enforcement, which is a lot broader than just security and compliance, and cost management, which similarly is not covered by the proposed name).  I suggest that the SIG Chairs and TOC Liaison fully describe the scope during the chartering process, and then engage fully in the debate around a suitable name to cover that scope.

Reply:
@quinton-hoole-2 I totally understand, and our group does want to reflect on this further during the chartering process. We agreed at our last meeting, however, to put in a request to change it in the initial document to better reflect our current understanding of what the group intends to do.

Pinging group members @dshaw @ultrasaurus @pragashj @deva @kbpawlowski @izgeri @hannibalhuang @jasonmelo @tsandall @sreetummidi @ckemper67 @rcolline @duglin @heavypackets @justincormack @lizrice @erikstmartin @quiqie @ericavonb @knowlengr @rae42 @rachelmyers @evan2645 @anweiss @tk2929 @goldberg10 @sublimino @iteration1 @chasemp @xuanjia @morellonet @alban @schu to add your +1 or -1 to this suggestion.

Reply:
Personally I'm a lot happier with the proposed change to Security and Compliance - it's a lot closer to my perception of what the group intends to do. Even if it's a placeholder that might be refined in future, it avoids likely confusion with "governance" in what I believe to be the normal open source sense i.e. project governance.

Reply:
As a way to unstick this a bit, I've taken process described by @quinton-hoole-2  and put it in a README, see https://github.com/quinton-hoole-2/toc/pull/3

In order to actually write down a process, I needed to fill in a few gaps and attempted to imagine something everyone would find to be reasonable.

Reply:
ok